Framework/DataHandling/test/LoadTest.h

```diff
@@ -233,6 +254,31 @@ class LoadTest : public CxxTest::TestSuite {
                boost::algorithm::ends_with(second, "CSP79590.raw"));
   }
 
+  /*
+   * This test loads and sums 2 IN4 runs from ILL
+   * without instrument prefix in the file names.
+   */
+  void test_ILLLoadMultipleFilesNoPrefix() {
+
+    ConfigService::Instance().setString("default.instrument", "IN4");
+    ConfigService::Instance().appendDataSearchSubDir("ILL/IN4/");
+
+    Load loader;
+    loader.initialize();
+    loader.setPropertyValue("Filename", "084446+084447.nxs");
+
+    std::string outputWS = "LoadTest_out";
+    loader.setPropertyValue("OutputWorkspace", outputWS);
+    TS_ASSERT_THROWS_NOTHING(loader.execute());
+
+    MatrixWorkspace_sptr output =
+        AnalysisDataService::Instance().retrieveWS<MatrixWorkspace>(outputWS);
+    MatrixWorkspace_sptr output2D =
+        boost::dynamic_pointer_cast<MatrixWorkspace>(output);
+
+    TS_ASSERT_EQUALS(output2D->getNumberHistograms(), 397);
```

Comment:
In this type of tests where settings like the search path, default instrument, facility, etc. in the config service are changed, I tend to like the test better if it resets those values to their defaults. In large `Test.h` files there could be some confusion. For example, if there is a clash of file names, etc. between different test cases, or if other tests were assuming different defaults.